source/NVDAObjects/IAccessible/ia2Web.py

```diff
@@ -105,6 +105,10 @@ class BlockQuote(Ia2Web):
 	role = controlTypes.ROLE_BLOCKQUOTE
 
 
+class Treegrid(Ia2Web):
+	role = controlTypes.ROLE_TABLE

```

Comment:
Probably missed my earlier comment, but have you considered using ROLE_DATAGRID for these?

Reply:
Perhaps we could use that role, but a lot of other checks would need to change within the code I think. Users are used to the concept of a "table" in browse mode, and also normal ARIA grids (tables with focusable cells but not collapsable rows) also get exposed as tables at the moment.